docs/debugging.md

```diff
@@ -0,0 +1,45 @@
+# Debugging
+
+Ability to debug is important for apps developers. App SDK provides opt-in verbose logs that can be used
```

Comment:
```suggestion
The ability to debug is important for app developers. App SDK provides opt-in verbose logs that can be used.
```

Also, the sentence:
> App SDK provides opt-in verbose logs that can be used

reads like it was cut short. Is there supposed to be something after "can be used"? Maybe it needs to be rephrased.

Reply:
Rephrased + applied all suggestions